keysign/QRCode.py

```diff
@@ -317,5 +318,8 @@ def on_released(widget, event):
 if __name__ == '__main__':
     import sys
     logging.basicConfig(level=logging.DEBUG)
-    data = sys.argv[1]
+    if len(sys.argv) > 1:
+	    data = sys.argv[1]
```

Comment:
Be careful with the indentation. With Python we follow as much as possible the PEP 8, so we use 4 spaces.
Here you mixes tabs and spaces.

Reply:
That would be my editor.